design/one-pager-prevent-unnecessary-reconciles.md

```diff
@@ -0,0 +1,53 @@
+# Prevent unnecessary reconciles
+* Owner: Christian Artin (@gravufo)
+* Reviewers: Crossplane Maintainers
+* Status: Speculative
+
+## Introduction
+
+Today, Crossplane providers will reconcile resources every time there is a change, regardless of the change, _and_ at pod startup _and_ when the `SyncInterval` is hit.
```

Comment:
Nit: We wrap our design docs at 80 chars to make the markdown form a little more readable. Please do that - there's a bunch of editor extensions to format automatically.